### PR TITLE
Add tessellate server

### LINKED ISSUE #32: Tessellate web-service package
* Extract tessellate web-service logic (koa app) into a separate package.
* Name the package tessellate-server.
* ~~Use tessellate-server in fragment and bundler (and viewer and editor).~~

### PR DESCRIPTION
**Pull request by mfellner**
Closes #32, #9

### CODE REVIEW

**Comment by danpersa** [comment]
👍

**Comment by danpersa** [comment]
👍

File: packages/tessellate-server/src/nconf.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+function readFile(file: string, parser: (s: string) => Object): Object {
+  try {
+    return parser(fs.readFileSync(file, 'utf8'))
+  } catch (e) {
+    console.error(`Unable to read ${file}`)
+    return {}
+  }
+}
+
+function readJSONFile(file: string): Object {
```
Comment (semonte) [inline packages/tessellate-server/src/nconf.js]: For consistency we should not capitalize JSON, so `readJSONFile()` -> `readJsonFile().

Comment (mfellner) [inline packages/tessellate-server/src/nconf.js]: Done!

**Comment by semonte** [comment]
👍

**Pull request merged**